test/database/dao/BookingDataDAOTest.java

```diff
@@ -17,8 +20,54 @@ public class BookingDataDAOTest {
 
 	@Test
 	public void testSaveBooking() throws SQLException {
-		BookingDataDTO booking=new BookingDataDTO(LocalDateTime.now(),LocalDateTime.of(2023,12,12,10,30),PaymentMethodDTO.CASH,new BigDecimal("6000"));
+		LocalDateTime entryDate=LocalDateTime.now().withNano(0);
+		
+		BookingDataDTO booking=new BookingDataDTO(entryDate,LocalDateTime.of(2023,12,12,10,30),PaymentMethodDTO.CASH,new BigDecimal("6000.0"));
+		
+		BookingDataDTO bookingSaved = bookingDataDAO.save(booking);
+		Assert.assertNotNull(bookingSaved);
+		Assert.assertNotNull(bookingSaved.getId());
+		
+		BookingDataDTO bookingSearched = bookingDataDAO.searchByIdBooking(bookingSaved.getId());
+		Assert.assertNotNull(bookingSearched);
+		Assert.assertEquals(bookingSaved.getId(), bookingSearched.getId());
+		Assert.assertEquals(bookingSaved.getEntryDate(), bookingSearched.getEntryDate());
+		Assert.assertEquals(bookingSaved.getDepartureDate(), bookingSearched.getDepartureDate());
+		Assert.assertEquals(bookingSaved.getPrice(), bookingSearched.getPrice());
+		Assert.assertEquals(bookingSaved.getPaymentMethod(), bookingSearched.getPaymentMethod());
+		
+		BookingDataDTO bookingToModify=new BookingDataDTO(bookingSearched.getId(),LocalDateTime.of(2023,12,10,10,30), LocalDateTime.of(2024,01,01,10,30), new BigDecimal("30000"), PaymentMethodDTO.CREDIT);
+		
+		int bookingModified = bookingDataDAO.modify(bookingToModify);
+		Assert.assertNotNull(bookingSearched);
+		Assert.assertTrue(1 == bookingModified);
```

Comment:
Please make a select from the database and make sure the modified values were actually well modified.